src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -249,6 +250,11 @@ All parameters are case-insensitive.</value>
 
 -WindowStyle | -w
     Sets the window style to Normal, Minimized, Maximized or Hidden.
+
+-WorkingDirectory | -wd
```

Comment:
Looks like `-wo` should also be mentioned in the help text.

Reply:
The pwsh parameters work the same as PowerShell parameters in that if there is sufficient difference in the name, it will figure it out.  So `-wi` will work for `-windowstyle` and `-wo` will work for `-workingdirectory`, although `-w` is reserved for `-windowstyle`.  So for that reason, I don't think it needs to be added explicitly to the help.